ops/charm.py

```diff
@@ -240,7 +241,7 @@ class StartEvent(HookEvent):
     """Event triggered immediately after first configuration change.
 
     This event is triggered immediately after the first
-    :class:`ConfigChangedEvent`. Callback methods bound to the event should be
+    :class:`~ops.ConfigChangedEvent`. Callback methods bound to the event should be
```

Comment:
Scenario copies various docstrings from ops objects, particularly the statuses and the events. When there's something like `:class:ConfigChangedEvent` in the Scenario objects and Sphinx does `autoclass`, it can't find the `ConfigChangedEvent` object, so fails. With the leading `ops.` Sphinx knows where to find them even when the docstring is being used outside of ops. The `~` tells Sphinx to just display the name and not the `ops.`.

The docstring copying code could try to be smarter and auto-fix these (it's probably just replacing ":class:" with "class:ops." (and the backticks), with maybe some false positives to work around), or Scenario could have its own docstrings (maybe just saying "go read the ops class for more details").

Discussed this in a call and Ben's ok with leaving them as-is for the moment, now that it's clearer what they are there for.